test/tests/functional/pbs_qmgr.py

```diff
@@ -43,6 +43,7 @@
 
 
 class TestQmgr(TestFunctional):
+
```

Comment:
why the extra line here? Is that a convention?

Reply:
I think it was added by autopep8